packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -574,8 +588,16 @@ export class EditManager<
 		}
 	}
 
-	private getCommitSequenceId(commit: GraphCommit<TChangeset>): SequenceId {
-		return this.trunkMetadata.get(commit.revision)?.sequenceId ?? minimumPossibleSequenceId;
+	private getCommitSequenceId(trunkCommitOrTrunkBase: GraphCommit<TChangeset>): SequenceId {
+		const id = this.trunkMetadata.get(trunkCommitOrTrunkBase.revision)?.sequenceId;
+		if (id === undefined) {
+			assert(
```

Comment:
Not strictly related to this PR, but I added this assert when debugging and think we should keep it. We don't want to return the `minimumPossibleSequenceId` by _accident_ (rather we only want to return it for the trunk base commit) - if we did, it'd probably be hard to debug.